docs/tutorials/lets_write_a_test.md

```diff
@@ -1,74 +1,81 @@
+Make a file named `install.rb` and put the following code into it:
 
-    $ mkdir setup
-    $ cd setup
-    $ cat > install.rb << RUBY
-    test_name "Installing Puppet Enterprise" do
-    install_pe
-    RUBY
-    $ cd ..
-
-This places our install steps in a ruby script (install.rb) which will run on localhost, but the #install_pe method knows where to install puppet enterprise based on the host configuration in use.
-The install.rb script is used in our commandline to beaker, below.
-
-## Create a test file
-
-We need to create a test file to run.
-
-### Define some test commands to run
-#### Restart mcollective twice
-
-Here's our magic command that restarts mcollective:
-
-    restart_command = "bash -c '[[ -x /etc/init.d/pe-mcollective ]] && /etc/init.d/pe-mcollective restart'"
-
-#### Check to see if more than one mcollective process is running
+```ruby
+test_name "Installing and runnning HTTPD" do
+
+	step "Make sure HTTPD is not installed" do
+	  hosts.each do |host|
+		# Check if HTTPD is not installed
+		assert_equal(false, check_for_package(host, 'httpd'))
+	  end
+	end
+
+	step "Install HTTPD" do
+	  hosts.each do |host|
+		# Install HTTPD
+		install_package(host, 'httpd')
+	  end
+	end
+
+	step "Start HTTPD" do
+	  hosts.each do |host|
+		# Start HTTPD service
+		on(host, "service httpd start")
+	  end
+	end
 
-Here's our magic command that throws an error if more than one mcollective process is running:
+end
+```
 
-    process_count_check = "bash -c '[[ $(ps auxww | grep [m]collectived | wc -l) -eq 1 ]]'"
+This places our install steps in a ruby script (`install.rb`) which will run on localhost. The install.rb script is used in our commandline to beaker, below.
 
-### Put it all together
+## Create a test file
 
-Here's the finished acceptance test.
+Lets create test file that tests if HTTPD is installed and running on our hosts. Make a file called `mytest.rb` and add the following code to it:
 
 ```ruby
-test_name "/etc/init.d/pe-mcollective restart check"
-
-# Don't run these tests on the following platforms
-confine :except, :platform => 'solaris'
-confine :except, :platform => 'windows'
-confine :except, :platform => 'aix'
-
-step "Make sure the service restarts properly"
-hosts.each do |host|
-  # Commands to execute on the target system.
-  restart_command = "bash -c '[[ -x /etc/init.d/pe-mcollective ]] && /etc/init.d/pe-mcollective restart'"
-  process_count_check = "bash -c '[[ $(ps auxww | grep [m]collectived | wc -l) -eq 1 ]]'"
-
-  # Restart once
-  on(host, restart_command) { assert_equal(0, exit_code) }
-  # Restart again
-  on(host, restart_command) { assert_equal(0, exit_code) }
-  # Check to make sure only one process is running
-  on(host, process_count_check) { assert_equal(0, exit_code) }
+test_name "Check if HTTPD is installed and running" do
+
+  # Don't run these tests on the following platform
+  confine :except, :platform => 'windows'
```

Comment:
Not sure if this should be addressed in your `install.rb` for earlier feedback about SUTs not being suitable.